### ISSUE TITLE
Integrating with external Django project

### ISSUE THREAD

**Issue by AlmogCohen** [openwisp/openwisp-users#24]
While trying to follow the regular README for integrating into my existing Django app (which is based on Django 2.0) I had two problems. I'll write the solution . Let me know if you think it should be added to the repo README.

1. First I got weird whenever I tried to use `manage.py`:

```
SystemCheckError: System check identified some issues:

ERRORS:
<class 'openwisp_users.admin.EmailAddressInline'>: (admin.E202) 'account.EmailAddress' has no ForeignKey to 'openwisp_users.User'.
<class 'openwisp_users.admin.OrganizationUserInline'>: (admin.E202) 'openwisp_users.OrganizationUser' has no ForeignKey to 'openwisp_users.User'.
auth.User.groups: (fields.E304) Reverse accessor for 'User.groups' clashes with reverse accessor for 'User.groups'.
	HINT: Add or change a related_name argument to the definition for 'User.groups' or 'User.groups'.
auth.User.user_permissions: (fields.E304) Reverse accessor for 'User.user_permissions' clashes with reverse accessor for 'User.user_permissions'.
	HINT: Add or change a related_name argument to the definition for 'User.user_permissions' or 'User.user_permissions'.
openwisp_users.User.groups: (fields.E304) Reverse accessor for 'User.groups' clashes with reverse accessor for 'User.groups'.
	HINT: Add or change a related_name argument to the definition for 'User.groups' or 'User.groups'.
openwisp_users.User.user_permissions: (fields.E304) Reverse accessor for 'User.user_permissions' clashes with reverse accessor for 'User.user_permissions'.
	HINT: Add or change a related_name argument to the definition for 'User.user_permissions' or 'User.user_permissions'.
```
**Solution**: Adding this to the `settings.py`:
    
    AUTH_USER_MODEL = 'openwisp_users.User'
(I found it in your tests by the way....)

2. Then, I was unable to login to `/admin` due to an error:
```
    Site matching query does not exist.
```
**Solution**: Adding this to the `settings.py`:

    SITE_ID = 1

I guess that if someone is new to `allauth` they should check their documentation to take full advantage of the nice module you created. Their installation guide helps to set the app the is basing itself on yours. Check it out [here](http://django-allauth.readthedocs.io/en/latest/installation.html)

**Comment by nemesisdesign** [comment]
@AlmogCohen thanks for reporting this, could you send a patch to add these two additions to this section of the README? https://github.com/openwisp/openwisp-users#setup-integrate-in-an-existing-django-project

**Comment by nemesisdesign** [comment]
Closed by #25, thank you @AlmogCohen
